addons/udes_stock/tests/test_stock_move.py

```diff
@@ -107,3 +124,65 @@ def test04_unreserve_initial_demand(self):
         new_move_lines = MoveLine.search([("product_id", "=", self.fig.id)])
         self.assertEqual(move_lines, new_move_lines)
         self.assertIn(pack2_ml, new_move_lines)
+
+    def test05_pepare_and_create_single_move_line(self):
+        """ Prepare and create a single move line and check values are correct """
+        product_uom_qty = 5
+
+        move_values = self.Picking._prepare_move(self.pick, [self._pick_info])
+        move = self.Picking._create_move(move_values)
+
+        # Check the prepared move_line_values are correct
+        move_line_values = self.Move._prepare_move_line(move, product_uom_qty)
+        self.assertEqual(
+            move_line_values, self._get_expected_move_line_values(move, product_uom_qty)
+        )
+
+        # Create the move line
+        move_line = self.Move._create_move_line(move_line_values)
+
+        # Check that one move line has been created and that the move now contains
+        # the created move line
+        self.assertEqual(len(move_line), 1)
+        self.assertEqual(move.move_line_ids, move_line)
+
+    def test06_pepare_and_create_multiple_move_lines(self):
+        """ Prepare and create a multiple move lines and check values are correct """
+        apple_uom_qty = 5
```

Comment:
Same as test05.

Reply:
Have changed `banana_uom_qty` to 2. Apple didn't actually have any quants, so have created a quant set to `apple_uom_qty` to make the test a bit more realistic.

Reply:
Also confirmed the picking and assigned the stock at the end of the test.